test/correctness/compute_at_update_stage.cpp

```diff
@@ -0,0 +1,37 @@
+#include <Halide.h>
+
+using namespace Halide;
+
+int main(int argc, char* argv[]) {
+    Func g;
+
+    {
+        Func f;
+        Var x, y;
+
+        f(x, y) = x + y;
+
+        g(x, y) = 0;
+        g(x, y) += f(x, y);
+
+        g.update().reorder(y, x);
+        f.store_at(g, x).compute_at(g, y);
+    }
+
+    Target target = get_jit_target_from_environment();
+    Halide::Buffer<int> out = g.realize(10, 10, target);
```

Comment:
The default value for the target arg is this anyway, so just call g.realize(10, 10) and skip the local target variable.